### DISCUSSION TITLE
Design System / Project Philosophy

### DISCUSSION THREAD

**Discussion by boweihan** [rewindui/rewindui#78] [General]
First of all, awesome project! I see lots of excellent practices in this repo and a great start.

I started fiddling with creating some new components for the library (starting with a slider) and wanted to know if you intend for components to follow any common design system or style guide. Is there a design philosophy for pre-styling of components that contributors should be aware of?

**Comment by NickDunas** [comment]
I am glad that you like it @boweihan! I really thank you for your previous contribution and your interest in creating new components.

I must admit that I'm more of a developer than a designer, so my focus is on ensuring they look aesthetically pleasing and that the styles are consistent across components.

However, please don't stress too much about it. If you can handle the logic for a new component, I'll be more than happy to assist you with the styling as well!
